src/index.html

```diff
@@ -11,7 +11,161 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="form__fieldset">
+        <legend>Personal information</legend>
+
+        <label class="form__label">
+          Surname:
+          <input 
+            type="text" 
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form__label">
+          Name:
+          <input 
+            type="text" 
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form__label">
+          How old are You?
+          <input 
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="form__label">
+          Full date of birth:
+          <input 
+            type="date"
+            name="dateOfBirth"
+            required
+          >
+        </label>
+
+        <label class="form__label">
+          I accept the term of agreement
+          <input type="checkbox">
+        </label>
+      </fieldset>
+
+      <fieldset class="form__fieldset">
+        <legend>Registration</legend>
+
+        <label class="form__label">
+          E-mail:
+          <input 
+            type="email" 
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="form__label">
+          Password:
+          <input 
+            type="password" 
+            name="password"
+            minlength="8"
+            maxlength="32"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form__fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="form__label">
+          Do you love cats?
+          <input 
+            type="radio" 
+            name="doLoveCats"
+            value="yes"
+          >Yes
+          <label>
+            <input 
+              type="radio" 
+              name="doLoveCats"
+              value="no"
+            >No
+          </label>  
+        </label>
```

Comment:
```suggestion
        <div class="form__label">
          Do you love cats?
          <input 
            type="radio" 
            name="doLoveCats"
            value="yes"
          >Yes
          <label>
            <input 
              type="radio" 
              name="doLoveCats"
              value="no"
            >No
          </label>  
        </div>
```

because when I click on `No` text I expecting `input` with value `no` should be checked